src/query/storage/block.go

```diff
@@ -115,7 +115,10 @@ func (m *multiBlockWrapper) WithMetadata(
 	}
 
 	if m.unconsolidated != nil {
-		unconsolidated, err = m.unconsolidated.WithMetadata(meta, seriesMetas)
+		unconsolidated, err = m.unconsolidated.WithMetadata(
```

Comment:
what's your line limit set ? I think two parameters on one line are fine, splitting everything out hurts readability a bit

Reply:
Really? Find this a bit easier to read personally

Reply:
okay, I can understand that you want to write code this way but can you avoid reformatting code you're not changing ?

Reply:
This was changed earlier before the revert